android/KMEA/app/build.gradle

```
@@ -55,17 +64,19 @@ android {
 
 dependencies {
     // 1.6.0-rc01 needed to resolve https://issuetracker.google.com/issues/238425626
-    implementation 'androidx.appcompat:appcompat:1.6.1'
+    implementation 'androidx.appcompat:appcompat:1.7.0'
     // material:1.7.0 will need Gradle plugin 7.1.0+
     implementation 'com.google.android.material:material:1.12.0'
-    implementation 'commons-io:commons-io:2.13.0'
+    implementation 'commons-io:commons-io:2.16.1'
     implementation 'io.sentry:sentry-android:7.8.0'
     implementation 'androidx.preference:preference:1.2.1'
 
     // Robolectric
-    testImplementation 'androidx.test:core:1.5.0'
-    testImplementation 'androidx.test.ext:junit:1.1.5'
-    testImplementation 'org.robolectric:robolectric:4.10.3'
+    testImplementation 'androidx.test.ext:junit:1.2.1'
+    testImplementation 'org.robolectric:robolectric:4.14.1'
+    testImplementation 'androidx.test:core:1.6.1'
+    testImplementation 'androidx.test.ext:junit:1.2.1'
+    testImplementation 'org.robolectric:robolectric:4.14.1'
```

```suggestion
    testImplementation 'androidx.test.ext:junit:1.2.1'
    testImplementation 'org.robolectric:robolectric:4.14.1'
    testImplementation 'androidx.test:core:1.6.1'
```

No need for duplicates, right?

---

good catch